spec/support/system_spec_methods/login_support.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+module LoginSupport
+  def login_as(user)
+    visit new_user_session_path
+
+    fill_in 'Eメール', with: user.email
+    fill_in 'パスワード', with: user.password
```

Comment:
passwordが取得できるのはFactoryBot経由で作成したユーザーだけです。
たとえば、`user = User.find_by name: 'ken'` みたいな方法でユーザを取得するとpasswordはnilになります。
これはパスワードは不可逆な方式で暗号化されるためです。
（パスワードは `encrypted_password` というカラムに暗号化されて保存される）

というわけで、どんなユーザでも引数で渡せるとは限らないので、パスワードは事前にわかっているものを決め打ちで書く方が良いです。

```suggestion
    fill_in 'パスワード', with: "pass1234"
```

引数で渡せるようにしておくのも一手ですね。

```ruby
  def login_as(user, password = "pass1234")
    # ...
    fill_in 'パスワード', with: password
```

Reply:
修正しました